CMS/settings/base.py

```diff
@@ -51,7 +51,7 @@
     'home',
     'search',
     'content',
-    'coursefinder',
+    'coursefinder.apps.CoursefinderConfig',
```

Comment:
Should the other apps be configured in this way (home, search, content) i.e., a dotted path to something subclassing  AppConfig?

Reply:
Only changed this one as it needed to be set up this way to register the template tags. But agree for consistency they should all be the same.